packages/components/src/spinner/index.js

```diff
@@ -1,8 +1,54 @@
+/**
+ * External dependencies
+ */
+import styled from '@emotion/styled';
+
 /**
  * Internal dependencies
  */
-import { StyledSpinner } from './styles/spinner-styles';
+import { COLORS, CONFIG } from '../utils';
+
+/**
+ * WordPress dependencies
+ */
+import { SVG, Circle } from '@wordpress/primitives';
+
+// A dedicated wrapper allows us to apply width and shadow color based on config values
+// The SVG can only have a centered stroke so a stacked box-shadow accomplishes a matching 1.5px border
+export const StyledSpinner = styled.span`
+	width: ${ CONFIG.spinnerSize }px;
+	height: ${ CONFIG.spinnerSize }px;
+	box-shadow: inset 0 0 0 0.75px ${ COLORS.gray[ 300 ] },
+		0 0 0 0.75px ${ COLORS.gray[ 300 ] };
+`;
 
 export default function Spinner() {
-	return <StyledSpinner className="components-spinner" />;
+	const viewBox = `0 0 ${ CONFIG.spinnerSize } ${ CONFIG.spinnerSize }`;
+	const radius = Number( CONFIG.spinnerSize ) / 2;
+	const strokeDasharray = [
+		Number( CONFIG.spinnerSize ),
+		Number( CONFIG.spinnerSize ) * 10,
+	];
+
+	return (
+		<StyledSpinner className="components-spinner">
+			<SVG
+				viewBox={ viewBox }
+				xmlns="http://www.w3.org/2000/svg"
+				className="components-spinner__track"
+				role="img"
+				aria-hidden="true"
+				focusable="false"
```

Comment:
Why is this attribute necessary? SVG elements shouldn't be focusable by default

Reply:
I'm fairily sure we've been adding `focusable="false"` because IE11 would focus SVGs regardless. Since we don't support IE11 anymore, we can remove it, however I wonder if we shouldn't do that in a separate PR, and remove across all SVGs in the codebase? Not a strong opinion.

Reply:
Thank you for the additional context!

> I wonder if we shouldn't do that in a separate PR, and remove across all SVGs in the codebase?

Makes sense to me. Let's tackle this separately

Reply:
Similar to my comment on a11y, let's keep this PR focussed on the visual stuff and address these details separately?